### PR TITLE
Run CI for windows

### PR DESCRIPTION
**Pull request by ybiquitous**
Split from #103

Currently, the behavior on macOS has many problems (also on CI). So I created this PR to focus on passing CI on Windows.

### CODE REVIEW

File: src/test/suite/index.ts [hunk 1]
```diff
@@ -11,7 +11,7 @@ export function run(): Promise<void> {
 
     const testsRoot = path.resolve(__dirname, '..');
 
-    mocha.timeout('30000');
+    mocha.timeout(60000);
```
Comment (ybiquitous) [inline src/test/suite/index.ts:14]: [note] 30,000 seconds seem too short on Windows (sometimes failed).

**Review by mame — APPROVED** [review]

File: .github/workflows/vscode.yml [hunk 2]
```diff
@@ -14,30 +14,38 @@ on:
...
         # TODO: support other platform such as macos
-        os: [ubuntu-latest]
+        os: [ubuntu-latest, windows-latest]
         ruby: ['3.1.0']
     runs-on: ${{ matrix.os }}
     steps:
     - name: Checkout
       uses: actions/checkout@v3
     - name: Set up Ruby
+      id: setup-ruby
```
Comment (ono-max) [inline .github/workflows/vscode.yml:27]: Why do we need to set `id` here?

Comment (ybiquitous) [reply .github/workflows/vscode.yml:27]: Ah, it's for `steps.setup-ruby.outputs.ruby-prefix`. I just copy-and-pasted the file from #103. 😅 

```yaml
env:
  RUBY_BIN_PATH: ${{ runner.os == 'macOS' && format('{0}/{1}', steps.setup-ruby.outputs.ruby-prefix, 'bin') || '' }}
```

https://github.com/ruby/vscode-typeprof/pull/103/files#diff-731c6f95e51cf7fd242dc335a02c96b18fd955089cfcfdb5c32cad3e884e02ecR49

Should I remove it?

Comment (ono-max) [reply .github/workflows/vscode.yml:27]: Ah, that's fine.

**Comment by ono-max** [comment]
Thank you for your work!

**Comment by ybiquitous** [comment]
@ono-max Thanks for the merge! But unfortunately, the CI job on `master` failed. I'm looking into the cause. 👀 

https://github.com/ruby/vscode-typeprof/actions/runs/5138725523/jobs/9248353039#step:9:73

**Pull request merged**